src/Symfony/Bridge/Doctrine/composer.json

```
@@ -47,6 +47,7 @@
         "doctrine/orm": "^2.6.3"
     },
     "conflict": {
+        "doctrine/orm": ">=3",
```

```suggestion
        "doctrine/orm": "<2.6.3|>=3",
```

I feel we should define the conflict in both directions.